scripts/install_obsolete_jsoncpp_1_7_4.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+set -e
+
+TEMPDIR=$(mktemp -d)
+JSONCPP="jsoncpp-1.7.4"
+
+cd $TEMPDIR
+wget -O "$JSONCPP.tar.gz" https://github.com/open-source-parsers/jsoncpp/archive/1.7.4.tar.gz
+
+tar xvzf "${JSONCPP}.tar.gz"
```

Comment:
Also there's no need to use a different filename, one could just use `1.7.4.tar.gz` as that will be the one after wget downloads it.